lib/jekyll/version.rb

```diff
@@ -1,3 +1,3 @@
 module Jekyll
-  VERSION = '3.1.6'
+  VERSION = "3.1.6".freeze
```

Comment:
Leave as-is. When we do the magic comment for 2.3.0 and we drop support for <= 2.3.0, we'll remove this :smiel